src/bun.js/ConsoleObject.zig

```diff
@@ -921,6 +921,7 @@ pub const Formatter = struct {
         Event,
 
         Getter,
+        CustomGetter,
```

Comment:
```suggestion
        GetterSetter,
        CustomGetterSetter,
```
If we're making changes it probably makes sense to change these names to be more descriptive

Reply:
Agreed, thanks!